src/Apps/Order/Routes/Status/index.tsx

```diff
@@ -142,6 +142,7 @@ const offerMessages = {
           Thank you for your response. The seller will be informed of your
           decision to end the negotiation process.
         </p>
+        <br />
```

Comment:
For some reason there's a space in Storybook but this `br` is needed here to create a space when running the full app. In Storybook, this will appear larger than it will appear in the app.

Reply:
I looked into this. Indeed force is injecting extra css. It's from a file called `main_layout.styl`. Not sure what to do about it, but whatever we do it should result in having things look identical in both storybook and force.

@damassi @zephraph could use your input on this. Does it make more sense to remove the global styles from the order app config in force, or copy them into reaction's storybook setup?

Reply:
Oof, that's a challenge. Removing it all together might not be good because we're not sure what it could effect. Copying the styles over to storybooks doesn't feel great, but it might be the best solution.